site/pages/store-locator.json

```diff
@@ -0,0 +1,9 @@
+{
+    "props": {
+        "locations": {
+            "filters": []
+        }
+    },
+    "template": "templates\/store\/locations",
+    "route": "s\/store-locator"
```

Comment:
do these need to be escaped? does it work if escaped?

Reply:
I think these were added automatically when I downloaded the files 🤔  will remove